### PR TITLE
Added German localization

### PR DESCRIPTION
**Pull request by LogischerJonasHD**
Fully localized and free from spelling mistakes!

### CODE REVIEW

**Comment by Sanduhr32** [comment]
German uses the present imperative more than past tense.

**Comment by Sanduhr32** [comment]
Still looks like a word for word translation and not German.

**Comment by LogischerJonasHD** [comment]
> Still looks like a word for word translation and not German.
@Sanduhr32

It certainly is German.
How about you actually give examples or suggest changes to certain lines instead of purely criticizing.
Like giving useful feedback.

File: src/main/resources/localization/messages-de.properties [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+info.help.creation = Giveaway Erstellungs Befehle
+info.help.manipulation = Giveaway Bedienungs Befehle
+info.help.settings = GiveawayBot Einstellungs Befehle
+giveaway.leave = Giveaway verlassen
+giveaway.ends = Endet
+giveaway.ended = Endete
+giveaway.hosted = Gehostet von
+giveaway.entries = Einträge
+giveaway.winners = Gewinner
+giveaway.summary = Giveaway Zusammenfassung
```
Comment (Sanduhr32) [inline src/main/resources/localization/messages-de.properties:77]:
```suggestion
error.permissions.user = Um diesen Befehl zu nutzen benötigst du die Berechtigung "Server verwalten".
error.permissions.perms_or_role = Um diesen Befehl zu nutzen benötigst du die Berechtigung "Server verwalten" oder die Verwaltungsrolle {0}.
error.permissions.perms_no_role = Um diesen Befehl zu nutzen benötigst du die Berechtigung "Server verwalten". Eine Verwaltungsrolle wurde noch nicht festgelegt. Ein Admin kann diese mit `{0}` festlegen.
error.permissions.bot = Mir fehlen die folgenden Berechtigungen in diesem Kanal: `Nachrichten lesen`, `Nachrichten senden`, `Nachrichtenverlauf anzeigen`, `Links einbetten`. Setze diese Berechtigungen in diesem Kanal für mich oder füge den Bot [hier]({0}) erneut hinzu.
error.invalid_time.format = Ich konnte `{0}` in keine gültige Zeit umwandeln!
error.invalid_time.min = Die angegebene Dauer ({0}) muss mindestens {1} betragen!
error.invalid_time.max = Die angegebene Dauer ({0}) darf maximal {1} betragen!
error.max_giveaways.guild = In diesem Server laufen bereits {0} Gewinnspiele!
error.max_giveaways.channel = In diesem Kanal laufen bereits {0} Gewinnspiele!
error.invalid_winners.format = Ich konnte `{0}` in keine gültige Anzahl an Gewinnern umwandeln!
error.invalid_winners.max = Die angegebene Anzahl an Gewinnern ({0}) befindet sich außerhalb von 1 und {1}!
error.invalid_prize.length = Preislänge muss innerhalb {0} Zeichen sein!
error.invalid_descr.length = Die Beschreibungslänge darf maximal {0} Zeichen betragen!
error.invalid_emoji.length = Der benutzerdefinierter Knopftext muss kürzer als {0} Zeichen sein!
error.invalid_emoji.choice = Vollständig benutzerdefinierte Knöpfe sind nur mit Premium verfügbar. Nutzer ohne Premium können folgende Emoji verwenden: {0}
error.invalid_id = Ich konnte `{0}` in keine Nachrichten ID umwandeln!
error.message_not_found = Ich konnte keine Nachricht mit der ID `{0}` in diesem Kanal finden!
error.invalid_message = Diese Nachricht ist kein beendetes Gewinnspiel!
error.giveaway_not_found = Ich konnte kein Gewinnspiel mit der ID `{0}` finden!
error.giveaway_already_entered = Du bist diesem Gewinnspiel bereits beigetreten!
error.giveaway_not_entered = Du bist nicht in diesem Gewinnspiel eingetragen!
error.no_dms = Dieser Bot kann nicht in Direktnachrichten verwendet werden!
error.giveaway_cooldown = Warte die Abklingzeit des Servers ab um weitere Gewinnspiele zu erstellen.
error.giveaway_ended = Das Gewinnspiel wurde bereits beendet. 
error.invalid_color = Ich konnte `{0}` in keine gültige Farbe konvertieren!
error.reroll_message_option = Gewinnspiele können auch direkt von der originalen Gewinnspiel-Nachricht aus über Rechtsklick (Desktop) / Drücken und halten (Mobile) > **Apps** > **Reroll Giveaway** neu gewürfelt werden.
error.generic.ending = Es ist ein Fehler beim Beenden des Gewinnspiels aufgetreten.
error.generic.delete = Das Gewinnspiel wurde gelöscht, aber die Discord Nachricht konnte nicht gelöscht werden.
error.generic.creation = Es ist ein Fehler beim Erstellen eines Gewinnspiels aufgetreten.
error.generic.enter = Es ist ein Fehler aufgetreten, dein Los konnte nicht validiert werden.
error.generic.reroll = Es ist ein Fehler beim erneuten Auslosen des Gewinnspiels aufgetreten.
error.generic = Es ist ein Fehler aufgetreten.
warning.no_giveaways = Zurzeit laufen keine Gewinnspiele!
warning.not_supported = Diese Funktion wird momentan nicht unterstützt.
warning.no_entries = Keine gültigen Lose, also konnte kein Gewinner bestimmt werden!
success.giveaway_created = Das Gewinnspiel wurde erfolgreich erstellt! ID: {0}
success.entered = Du hast dein Gewinnspiel-Los gezogen.
success.leave = Du hast das Gewinnspiel-Los zurückgegeben.
success.ping = Pong!
success.winner = Herzlichen Glückwunsch {0}! Du hast **{1}** gewonnen!
success.settings.role = Alle Benutzer mit der {0} Rolle können jetzt Gewinnspiele verwalten!
success.settings.color = Die Einbettungsfarbe für dieses Gewinnspiele wurde zu `{0}` geändert.
success.settings.emoji = Der benutzerdefinierte Emoji-Knopf wurde zu `{0}` geändert.
success.giveaway_ended = Gewinnspiel {0} erfolgreich beendet!
success.giveaway_reroll = {0} hat das Gewinnspiel erneut ausgelost! Herzlichen Glückwunsch {1}!
success.giveaway_delete = Gewinnspiel {0} erfolgreich gelöscht!
info.invite = Hallo! Ich bin **GiveawayBot**! und ich erleichtere es Gewinnspiele in deinem Discord Server schnell und reibungslos zu halten! \
\nDu kannst mich mit diesem Link zu deinem Server hinzufügen:
info.about = Informationen / Statistiken über
info.about.brief = Halte Gewinnspiele schnell und einfach!
info.about.long = Hallo! Ich bin **GiveawayBot**  und ich erleichtere es Gewinnspiele in deinem Discord Server zu halten! Ich wurde von [**jagrosh**#4824](https://jagrosh.com/) (<@113156185389092864>) entwickelt. Alle meine Befehle beginnen mit `/g`. Vergiss nicht, meine Webseite **https://giveawaybot.party/** anzuschauen!
info.about.stats = Statistiken
info.about.stats.servers = {0} Server
info.about.stats.giveaways = {0} Gewinnspiele
info.about.links = Links
info.about.links.website = Webseite
info.about.links.invite = Einladung
info.about.links.support = Hilfe
info.settings = Einstellungen
info.settings.owner = Serverbesitzer
info.settings.premium = Premium Status
info.settings.emoji = Gewinnspiel Emoji
info.settings.role = Verwaltungsrolle
info.settings.color = Einbettungsfarbe
info.settings.locale = Sprache
info.settings.etc = Um festzulegen, welche Rollen Gewinnspiele verwalten können, gehe zu **Server Einstellungen** > **Integrationen** > **GiveawayBot**
info.help.general = Allgemeine Befehle
info.help.creation = Befehle zur Gewinnspiel-Erstellung
info.help.manipulation = Befehle um Gewinnspiele zu modifizieren
info.help.settings = GiveawayBot Konfigurationsbefehle
giveaway.leave = Gewinnspiel verlassen
giveaway.ends = Wird ausgelost
giveaway.ended = Wurde ausgelost
giveaway.hosted = Erstellt von
giveaway.entries = Einträge / Lose
giveaway.winners = Gewinner
giveaway.summary = Gewinnspiel Zusammenfassung
```

**Review by Sanduhr32 — COMMENTED** [review]
requested examples / suggested changes.

File: src/main/resources/localization/messages-de.properties (hunk 1 above)
Comment (LogischerJonasHD) [reply src/main/resources/localization/messages-de.properties:77]: I like the way you changed some lines entirely and gave it a completely different "feel" but I don't quite like the term "Gewinnspiel". Sure, in a way it's kind of just that but it's not pretty fitting, I think. After all, the Bot will still be called GiveawayBot and I guess everyone knows what a Giveaway is so thats why I didn't change this term and I'd like to keep it like that but I won't insist on that either.

I find it a bit weird that you added an article to line 13 but not to line 12 but I guess thats just an oversight.
I think most of the changes you made are working pretty good but you completely simplified line 24, which wasn't that bad before.
Anyways, thanks for your suggestions, I'll take care of them tomorrow!

**Comment by Sanduhr32** [comment]
Stein drüber 👍🏼

**Pull request merged**